src/api/interface.ts

```diff
@@ -102,34 +103,36 @@ declare class Channel {
    * Note that this function can be used without first joining the channel, allowing applications to broadcast on
    * channels that they aren't a member of.
    * 
-   * This broadcast will be received by all windows that are members of this channel, *except* for the window that
-   * makes the broadcast. This matches the behaviour of the top-level FDC3 `broadcast` function.
    * `Error` with a string from the `ChannelError` enumeration.
    */
   public broadcast(context: Context): Promise<void>;
 
   /**
    * Returns the last context that was broadcast on this channel. All channels initially have no context, until a 
-   * window is added to the channel and then broadcasts. If there is not yet any context on the channel, this method
-   * will return `null`. The context is also reset back into it's initial context-less state whenever a channel is
```

Comment:
@nkolba I have removed the second part - seems like an implementation detail to me. FDC3 has no concept of "windows", so it shouldn't be in there.